packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/index.js

```diff
@@ -41,6 +41,8 @@ export const PreviewBox = ({
 }) => {
   const { trackUsage } = useTracking();
   const previewRef = useRef(null);
+  const [cropImageReady, setCropImageReady] = useState(false);
```

Comment:
I am just being neat picky  but can you start boolean constant names by `is|does|has|should` ?